### PR TITLE
shared netid xfer warning, tests

### PR DESCRIPTION
**Pull request by mikeseibel**

### CODE REVIEW

File: endorsement/dao/notification.py [hunk 1]
```diff
@@ -310,6 +310,47 @@ def warn_endorsers(notice_level, lifetime):
                 pass
 
 
+def create_warn_shared_owner_message(owner_netid, endorsements):
+    context = {
```
Comment (ezturner) [inline endorsement/dao/notification.py]: You're checking that endorsements is not None and has a len > 0 below - I'd do that here or add an underscore to signify that this is a private method.

File: endorsement/dao/uwnetid_admin.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+def get_owner_for_shared_netid(netid):
+    """
+    Return admin resources
+    """
+    try:
+        for admin in get_admins_for_shared_netid(netid):
+            if admin.is_owner():
+                return admin.name
+    except DataFailureException as ex:
+        if ex.status == 404:
```
Comment (ezturner) [inline endorsement/dao/uwnetid_admin.py]: What's this case? NetID change?

Comment (mikeseibel) [reply endorsement/dao/uwnetid_admin.py]: shared netid ownership change

File: endorsement/dao/notification.py (hunk 1 above)
Comment (mikeseibel) [reply endorsement/dao/notification.py]: there's a few other procedures in that pattern that could be adjusted

File: endorsement/dao/uwnetid_admin.py (hunk 2 above)
Comment (ezturner) [reply endorsement/dao/uwnetid_admin.py]: Er, rather, I mean catching the 404 and passing

Comment (mikeseibel) [reply endorsement/dao/uwnetid_admin.py]: more a deletion than change, but yeah

**Pull request merged**